internal/component/prometheus/exporter/catchpoint/catchpoint.go

```diff
@@ -0,0 +1,51 @@
+package catchpoint
+
+import (
+	"github.com/grafana/alloy/internal/component"
+	"github.com/grafana/alloy/internal/component/prometheus/exporter"
+	"github.com/grafana/alloy/internal/featuregate"
+	"github.com/grafana/alloy/internal/static/integrations"
+	"github.com/grafana/alloy/internal/static/integrations/catchpoint_exporter"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:      "prometheus.exporter.catchpoint",
+		Stability: featuregate.StabilityGenerallyAvailable,
```

Comment:
I don't think that this new component should be GA right away. I think that you should set it to Experimental for now